### PR TITLE
Messages actions

### PR DESCRIPTION
**Pull request by diegolmello**
- [x] delete
- [x] edit
- [x] copy
- [x] permalink
- [x] quote
- [x] reply
- [x] starmessage
- [x] pinmessage

### CODE REVIEW

**Comment by rodrigok** [comment]
@diegolmello Lint is broken

**Comment by diegolmello** [comment]
Tap and hold on message to open actions.

screen shot 2017-11-17 at 15 07 36
screen shot 2017-11-17 at 15 08 01

File: app/containers/message/Card.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default class Cards extends React.PureComponent {
 					</Card>
 				</TouchableOpacity>
 				<PhotoModal
-					title={this.props.data.title}
+					title={this.props.data.title || 'test'}
```
Comment (ggazzo) [inline app/containers/message/Card.js]: 🤔

Comment (diegolmello) [reply app/containers/message/Card.js]: Just a workaround to fix PropTypes.string.isRequired.
It will be fixed later on #55 .

**Pull request closed without merging**